src/cliAction/installDaemon/handler.go

```diff
@@ -0,0 +1,26 @@
+package installDaemon
+
+import (
+	"github.com/zerops-io/zcli/src/daemonInstaller"
+)
+
+type Config struct {
```

Comment:
Empty config. Shouldn't we delete this?

Reply:
It's feature rather than bug. I've tried to maintain same pattern for every handler. And RunConfig was designed for interactive mode. But you're right, for now it's useless.

Reply:
I don't mind it staying there then, if you have your reasons for it to stay.